dist/Dockerfile.deploy/Dockerfile

```diff
@@ -1,6 +1,9 @@
 FROM quay.io/app-sre/cincinnati:builder AS builder
 # build
 COPY . .
+# copy git information for built crate
+COPY .git/ ./.git/
```

Comment:
Why is this required?  because when you copy a git repository in  the .git directory should get copied correctly in to `../.git/`.

Reply:
`COPY . .` doesn't copy hidden files or dirs